R/get_hub_locations.R

```diff
@@ -0,0 +1,31 @@
+#' Create data/hub_locations.rda for loading data from zoltar and covidcast
```

Comment:
I think we should remove this function and replace it by just calling the existing hub_locations dataset that is in the package already. If we need to add columns we can do that.